### PR TITLE
community/suricata: add rust support for more arches

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: community/suricata/APKBUILD [hunk 1]
```diff
@@ -24,7 +24,7 @@ _enable_luajit=""
 _enable_rust=""
 
 case "$CARCH" in
-	armhf|armv7|x86_64)
+	aarch64|armhf|armv7|ppc64le|x86|x86_64)
```
Comment (Cogitri) [inline community/suricata/APKBUILD]: I'm pretty sure luajit isn't actually functional on at least aarch64.

Comment (J0WI) [reply community/suricata/APKBUILD]: AFAIK it works on all but `s390x`

Comment (Cogitri) [reply community/suricata/APKBUILD]: Well, at least on my aarch64 machine it just instantly explodes

Comment (J0WI) [reply community/suricata/APKBUILD]: `luajit` or `suricata`?
Btw. there's a PR to address `s390x`: #7666

Comment (Cogitri) [reply community/suricata/APKBUILD]: luajit

**Comment by Cogitri** [comment]
7ecfea85438f2defe607195522d8807a50239370

**Pull request closed without merging**